tests/unit/meta/Drag.ts

```diff
@@ -54,7 +46,8 @@ registerSuite({
 		const widget = new TestWidget();
 		widget.append(div);
 
-		resolveRAF();
+		resolvers.resolve();
```

Comment:
why are you calling this twice here and only once elsewhere? It was only called once before.

Reply:
@tomdye this is because after the render we need to resolve the `requestIdleCallback`s, which with meta can cause an invalidation. So  means we need to resolve the `requestAnimationFrame` (and idle callbacks again) to reflect the outcome from the render.